otx/api/usecases/exportable_code/demo/requirements.txt

```diff
@@ -1,4 +1,4 @@
-openvino==2022.3.0
-openmodelzoo-modelapi==2022.3.0
-otx @ git+https://github.com/openvinotoolkit/training_extensions/@a70d8f486f2dab289dffb6879606a2dfd827964d#egg=otx
+openvino==2023.0
```

Comment:
@sovrasov Could you also check if there's a noticeable accuracy drop on instance segmentation? Recently someone reported this issue: https://github.com/openvinotoolkit/training_extensions/issues/2252#issue-1763838906. 

That being said, I still notice a large accuracy drop after installing 2023.0 with PyTorch 1.13.x. Could you verify if this is no longer an issue for PyTorch 1.13.x after the upgrade?

Reply:
@eugene123tw @jaegukhyun here are my results on coco_subset_cls_decr (496 imgs in val) from regression tests:
| Model  | pytorch 1.13 | OV2023 + new model API   |
|---|---|---|
| efficientnetb2b_maskrcnn  | 0.3488 | 0.3546  | 
| resnet50_maskrcnn  |  0.5212 | 0.5170  |